core/workflow-core/src/main/scala/edu/uci/ics/amber/util/IcebergUtil.scala

```diff
@@ -47,12 +47,16 @@ import java.sql.Timestamp
 import java.time.LocalDateTime
 import java.time.ZoneId
 import scala.jdk.CollectionConverters._
+import edu.uci.ics.texera.service.util.S3LargeBinaryManager
 
 /**
   * Util functions to interact with Iceberg Tables
   */
 object IcebergUtil {
 
+  // Special prefix to identify LARGE_BINARY type in Iceberg
+  val LARGE_BINARY_PREFIX = "LARGE_BINARY_URI:"
```

Comment:
rename to TEXERA_LARGE_BINARY_TYPE_ATTRIBUTE_NAME_PREFIX
change the value to "TEXERA_LARGE_BINARY:"

Reply:
also move this const to Attribute.java, as it may be used by other components, not just iceberg